README.md

```diff
@@ -26,7 +26,8 @@ LocalEGA is divided into several components, as docker containers.
 | ingesters   | Split the Crypt4GH header and move the remainder to the storage backend. No cryptographic task, nor access to the decryption keys. |
 | verifiers   | Decrypt the stored files and checksum them against their embedded checksum. |
 | vault       | Storage backend: as a regular file system or as a S3 object store. |
-| outgesters  | Front-facing check for download permissions. |
+| finalizers  | Handle the so-called _Stable ID_ filename mappings from CentralEGA. |
```

Comment:
This part can be merged with the other changes to the readme file

Reply:
maybe so, but that'll do it too